### ISSUE TITLE
About C3D

### ISSUE THREAD

**Issue by q-rong** [hobincar/pytorch-video-feature-extractor#3]
Hi, thank you for your awesome code. You are an excellent researcher.

In my research, I need to use `C3D` to extract features.
I noticed that you have removed `C3D`. Can you restore `C3D`  if possible? Or can you tell me the link to `C3D` that you have used?

Thank you very much. Looking forward to your reply.
Best wishes.

**Comment by hobincar** [comment]
Hi, thank you for visiting my repository.

As you mentioned, I tried to extract C3D features using the codes and the pre-trained weights from [here](https://github.com/DavideA/c3d-pytorch). However, when I use the features to [a video captioning model](https://github.com/hobincar/SA-LSTM), I got much lower scores than the reported ones. I could not find what the problem is and just removed the C3D codes from this repo.

If you want the code that I wrote, you can go straight to the point before the code was removed by simply typing
```
git checkout c91ff4045aefb0cf33b0f8567b1d714428f3a77c
```

Or you can see [the original pytorch code](https://github.com/DavideA/c3d-pytorch) that I referred to. But I recommend you to use [the original one](https://github.com/facebookarchive/C3D/tree/master/C3D-v1.1/examples/feature_extraction).

**Comment by q-rong** [comment]
Thank you for your help.

**Comment by Tramac** [comment]
> Hi, thank you for visiting my repository.
> 
> As you mentioned, I tried to extract C3D features using the codes and the pre-trained weights from [here](https://github.com/DavideA/c3d-pytorch). However, when I use the features to [a video captioning model](https://github.com/hobincar/SA-LSTM), I got much lower scores than the reported ones. I could not find what the problem is and just removed the C3D codes from this repo.
> 
> If you want the code that I wrote, you can go straight to the point before the code was removed by simply typing
> 
> ```
> git checkout c91ff4045aefb0cf33b0f8567b1d714428f3a77c
> ```
> 
> Or you can see [the original pytorch code](https://github.com/DavideA/c3d-pytorch) that I referred to. But I recommend you to use [the original one](https://github.com/facebookarchive/C3D/tree/master/C3D-v1.1/examples/feature_extraction).

Hi, maybe the pre-trained weights from [here](https://github.com/DavideA/c3d-pytorch) was trained on un-normalized data?
